cmd/protecodeExecuteScan.go

```diff
@@ -0,0 +1,317 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	pkgutil "github.com/GoogleContainerTools/container-diff/pkg/util"
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/protecode"
+	"github.com/google/go-containerregistry/pkg/legacy/tarball"
+	"github.com/google/go-containerregistry/pkg/name"
+)
+
+type protecodeData struct {
+	Target                               string `json:"target,omitempty"`
+	Mandatory                            bool   `json:"mandatory,omitempty"`
+	ProductID                            string `json:"productID,omitempty"`
+	ProtecodeServerURL                   string `json:"protecodeServerUrl,omitempty"`
+	ProtecodeFailOnSevereVulnerabilities bool   `json:"protecodeFailOnSevereVulnerabilities,omitempty"`
+	ProtecodeExcludeCVEs                 string `json:"protecodeExcludeCVEs,omitempty"`
+	Count                                string `json:"count,omitempty"`
+	Cvss2GreaterOrEqualSeven             string `json:"cvss2GreaterOrEqualSeven,omitempty"`
+	Cvss3GreaterOrEqualSeven             string `json:"cvss3GreaterOrEqualSeven,omitempty"`
+	ExcludedVulnerabilities              string `json:"excludedVulnerabilities,omitempty"`
+	TriagedVulnerabilities               string `json:"triagedVulnerabilities,omitempty"`
+	HistoricalVulnerabilities            string `json:"historicalVulnerabilities,omitempty"`
+}
+
+func protecodeExecuteScan(config protecodeExecuteScanOptions, influx *protecodeExecuteScanInflux) error {
+	c := command.Command{}
+	// reroute command output to loging framework
+	c.Stdout(log.Entry().Writer())
+	c.Stderr(log.Entry().Writer())
+
+	return runProtecodeScan(&config, influx)
+}
+
+func runProtecodeScan(config *protecodeExecuteScanOptions, influx *protecodeExecuteScanInflux) error {
+
+	//create client for sending api request
+	if config.Verbose {
```

Comment:
there is no need for checking `config.Verbose`.

`log.Entry().Debugf()` will do the job ...